x-pack/platform/plugins/shared/alerting/server/alerts_service/alerts_service.ts

```diff
@@ -432,6 +432,7 @@ export class AlertsService implements IAlertsService {
       const componentTemplate = getComponentTemplate({
         fieldMap: mappings.fieldMap,
         dynamic: mappings.dynamic,
+        dynamicTemplates: mappings.dynamicTemplates,
```

Comment:
i would add a unit test to the getComponent template function to verify this is passed through

Reply:
added a unit test